scripts/shell/init_seednode.sh

```
@@ -0,0 +1,29 @@
+
+# Create data directory (with parent directories if needed)
+sudo mkdir -p /data/seednode0/.treasurenetd
+
+# Remove old configuration in data directory if exists
+sudo rm -rf /data/seednode0/.treasurenetd
+
+# Move newly initialized node config to data directory
+sudo mv /home/ubuntu/.treasurenetd /data/seednode0/.treasurenetd
+
+# Change to main node's config directory
+cd /data/node0/.treasurenetd/config/
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)